lsp-fiber/test/lsp_fiber_test.ml

```diff
@@ -209,7 +209,7 @@ let%expect_test "ent to end run of lsp tests" =
   [%expect.unreachable]
   [@@expect.uncaught_exn
     {|
-  ("Abort: Ppx_yojson_conv_lib__Yojson_conv.Of_yojson_error(_, 870828711)")
+  ("Abort: Jsonrpc: json conversion failed: invalid structured value")
```

Comment:
I fixed this bug. So please rebase to remove this change.

Reply:
Done!